apisix/schema_def.lua

```diff
@@ -332,6 +332,10 @@ local upstream_schema = {
             description = "enable websocket for request",
             type        = "boolean"
         },
+        labels = {
+            description = "key/value pairs to specify attributes",
```

Comment:
Why we need a `key-value` table for labels? I think array object is simpler.

Reply:
I think `key-value` is more descriptive, and we can make some conventions on the `key`, which is more readable.
e.g.
```
lables: {
    "version": "v1"
}
```

Reply:
I think `key-value` table is ok. but we need to set some limit to it, such as `maxProperties`, item type and length ...

Reply:
@membphis I guess key-value may be much useful for marking/filtering/grouping objects, such as we could use the labels `{"env": "release"}` and `{"env": "testing"}` to distinguish the publishing stage for a Route.

Reply:
Hi @membphis @nic-chen @gxthrj  , PR is updated and the format of value in `labels` is limited to `string`, with set maxProperties to 16. A new test case which contains invalid value format is also added. Feel free to leave your comments. Thanks.

Reply:
ok, got it. many thx for your explain.